package.json

```diff
@@ -42,6 +42,7 @@
     "react-drag-drop-files": "^2.3.10",
     "react-icons": "^4.8.0",
     "react-pdf": "^6.2.2",
+    "react-popper": "^2.3.0",
```

Comment:
Doesn't ProseMirror support autocomplete? If so, could we use it for this use-case?